config/pipelineTemplate.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Joi = require('joi');
+const JobsSchema = require('./base').jobs;
+const Shared = require('./base').shared;
+const Parameters = require('./parameters');
+const Template = require('./template');
+
+const SCHEMA_CONFIG = Joi.object()
+    .keys({
+        jobs: JobsSchema.required(),
+        shared: Shared,
+        parameters: Parameters.parameters.default({})
+    })
+    .unknown(false);
+
+const SCHEMA_TEMPLATE = Joi.object()
+    .keys({
+        namespace: Template.namespace.required(),
+        name: Template.name.required(),
```

Comment:
We cannot use Job template schema for `name` as it allows the `name` to be prefixed with `namespace`.

```suggestion
        name: Joi.string()
        .regex(Regex.TEMPLATE_NAME_NO_SLASH)
        .description('Name of the template')
        .example('nodePipeline'),
        .required(),
```